data/src/test/java/org/apache/iceberg/data/DeleteReadTests.java

```diff
@@ -329,7 +329,7 @@ private StructLikeSet rowSetWithoutIds(int... idsToRemove) {
     return set;
   }
 
-  protected StructLikeSet rowSetWitIds(int... idsToRetain) {
+  protected StructLikeSet rowSetWithIds(int... idsToRetain) {
```

Comment:
Can you fix this in a separate PR? This file doesn't need to change and it could cause commit conflicts.